### PR TITLE
[DOCS-226] Notifications

### PR DESCRIPTION
**Pull request by ana-dashuk-cobalt**
### Added

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
| Manage Notifications | https://deploy-preview-295--cobalt-docs.netlify.app/platform-deep-dive/collaboration/manage-notifications/ |  |

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

### CODE REVIEW

File: content/en/Platform Deep Dive/Collaboration/manage-notifications.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+- Created or deleted a pentest
+- Posted an update for a pentest
+- Reported a vulnerability
+- Changed a finding state
+- Invited users to the organization
+
+{{% /expand %}}
+<br>
+
+Your [user role](/platform-deep-dive/collaboration/organization/user-roles/) determines which notifications you get.
```
Comment (ana-dashuk-cobalt) [inline content/en/Platform Deep Dive/Collaboration/manage-notifications.md:28]: > Your user role determines which notifications you get.

Not sure about this one.

File: content/en/Platform Deep Dive/Collaboration/manage-notifications.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+## Manage Pentest Notifications
+
+When you collaborate on a pentest, you get notifications for all associated events.
```
Comment (ana-dashuk-cobalt) [inline content/en/Platform Deep Dive/Collaboration/manage-notifications.md:32]: > When you collaborate on a pentest, you get notifications for all associated events.

Do users get notifications for **all** events? Should we say something like "for some associated events"?

File: content/en/Platform Deep Dive/Collaboration/manage-notifications.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    ![Manage notification preferences for a pentest](/deepdive/ManagePentestNotifications.png "Manage notification preferences for a pentest")
+
+## Manage Finding Notifications
+
+You start getting notifications for a finding when:
+
+- You submitted the finding.
+- Someone assigned you to the finding.
+- You left a comment on the finding.
+- Someone mentioned you in a comment.
```
Comment (ana-dashuk-cobalt) [inline content/en/Platform Deep Dive/Collaboration/manage-notifications.md:53]: I took these use cases from an internal Confluence page.

File: content/en/Platform Deep Dive/Collaboration/manage-notifications.md (hunk 2 above)
Comment (1roshan) [reply content/en/Platform Deep Dive/Collaboration/manage-notifications.md:32]: When you collaborate on a pentest, you get notifications dependent on your notification preferences.

Watching is generally ALL events
Participating is usually just mentions (e.g. @ana-dashuk-cobalt have a look at this finding) 
Ignoring = no events unless you explicitly follow a finding.

File: content/en/Platform Deep Dive/Collaboration/manage-notifications.md (hunk 3 above)
Comment (1roshan) [reply content/en/Platform Deep Dive/Collaboration/manage-notifications.md:53]: > You start getting notifications for a finding when: You left a comment on the finding.

I wouldn't get a notification if I commented on a finding - however, if "someone left a comment on the Finding", you would receive a notification while "Watching" the pentest. 

Participating would mean you may only receive assigning and mentions.

Comment (1roshan) [reply content/en/Platform Deep Dive/Collaboration/manage-notifications.md:53]: Also customers can't submit a finding, pentesters would create one.

Comment (ana-dashuk-cobalt) [reply content/en/Platform Deep Dive/Collaboration/manage-notifications.md]: @1roshan Thank you, I addressed your feedback.

Comment (ana-dashuk-cobalt) [reply content/en/Platform Deep Dive/Collaboration/manage-notifications.md]: Hi @1roshan, can we please finalize this pull request? I addressed your feedback and also changed the label for findings to **Follow**, based on the recent updates.

**Review by 1roshan — APPROVED** [review]
LGTM

**Pull request merged**